womtool/src/test/resources/validate/wdl_draft3/valid/cmd_whitespace_none/cmd_whitespace_none.wdl

```diff
@@ -0,0 +1,17 @@
+version 1.0
+
+workflow Test {
+
+  call Echo as echo
+
+}
+
+task Echo {
+
+  runtime { docker: "ubuntu:latest" }
+
+  command {
+echo "I am prefixed with nothing"
```

Comment:
Rather than just asserting "passes womification", we might be able to use the `WdlFileToWdlomSpec` to make assertions about what the final command line looks like.

... or even just unit test the `astToCommandSectionElement ` directly to avoid needing a surrounding framework altogether?

Reply:
Love the unit test idea, I'm not sure we have any coverage for the whitespace prefix normalization (only `gap_in_command.wdl` for dropping empy lines)

Reply:
The idea was much more appealing than reality proved to be; stuck to augmenting `WdlFileToWdlomSpec`